parlai/agents/rnn_baselines/agents.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+from parlai.core.agents import Agent
+from parlai.core.dict import DictionaryAgent
+from parlai.core.params import ParlaiParser
+from parlai.core.worlds import create_task
+
+from torch.autograd import Variable
+from torch import optim
+import torch.nn as nn
+import torch
+
+import copy
+import os
+import random
+import time
+
+
+class Seq2SeqAgent(Agent):
+    """Simple agent which uses an LSTM to process incoming text observations."""
+
+    @staticmethod
+    def add_cmdline_args(argparser):
+        argparser.add_arg('-hs', '--s2s-hiddensize', type=int, default=64,
+            help='size of the hidden layers and embeddings')
+        argparser.add_arg('-nl', '--s2s-numlayers', type=int, default=2,
+            help='number of hidden layers')
+        argparser.add_arg('-lr', '--s2s-learningrate', type=float, default=0.5,
+            help='learning rate')
+        argparser.add_arg('--cuda', action='store_true', default=False,
+            help='enable GPUs if available')
+        argparser.add_arg('--gpu', type=int, default=-1,
+            help='which GPU device to use')
+
+    def __init__(self, opt, shared=None):
```

Comment:
yeah this doesn't do that yet! I'd rather check it in before adding that functionality